Gemfile

```diff
@@ -1,8 +1,6 @@
 source 'https://rubygems.org'
 
 # Read the current Ruby version from the .ruby-version file
-ruby Bundler.read_file(Bundler.root.join('.ruby-version')).strip
```

Comment:
Why did this change? iirc the `Bundler.root` avoids issues when `bundle install` is invoked in a subdirectory.

If we're going to change it, it'd probably be best to use the more modern `ruby file: ".ruby-version"` [flavor](https://bundler.io/guides/gemfile_ruby.html).

Reply:
This fixes incompatibility with the VS Code Ruby extension, and still works in subdirectories. I can change to the new thing but I think it'll require a bump to bundler.